book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :book_rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @book_rentals = []
```

Comment:
- For better readability, the `book_rentals` could be simply called rentals. please, fix this to be `@rentals = [ ]`

Reply:
Done!